front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Destination.tsx

```diff
@@ -8,16 +9,23 @@ import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
 
 type DestinationProps = {
   zoomToFeaturePoint: (lngLat?: Position, id?: string) => void;
+  invalidPathItems?: string[];
 };
 
-const Destination = ({ zoomToFeaturePoint }: DestinationProps) => {
+const Destination = ({ zoomToFeaturePoint, invalidPathItems }: DestinationProps) => {
   const { getDestination } = useOsrdConfSelectors();
   const { updateDestination } = useOsrdConfActions();
   const destination = useSelector(getDestination);
 
   const dispatch = useDispatch();
   const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
 
+  const isInvalid =
+    invalidPathItems &&
+    destination &&
+    'trigram' in destination &&
+    invalidPathItems.includes(destination.trigram);
```

Comment:
Could we avoid using the `trigram` to identify path items? Path items all have an ID (note, _not_ the OP ID, which is stored separately in `operational_point`).